pyproject.toml

```diff
@@ -23,7 +23,7 @@ classifiers = [
     "Topic :: Internet :: WWW/HTTP",
     "Topic :: Software Development :: Libraries :: Python Modules",
 ]
-requires-python = "~=3.9"
+requires-python = ">=3.9,<4"
```

Comment:
Given that [`uv` maintainers are “generally very opposed to including upper bounds on the `requires-python` range”](https://github.com/astral-sh/uv/issues/14422#issuecomment-3045119395), I suggest we remove the upper bound here as well.